documentation/projects/planning.md

```diff
@@ -71,57 +71,36 @@ relevant step of the process is complete.
 
 ## Providing project updates
 
-At all stages of a project (except for `Not Started` projects), fortnightly
-public updates are the best way for Openverse contributors and maintainers to
-understand the health of a project. Importantly, these updates allow us to
-allocate more time or material resources to projects that need them before work
-stalls completely.
-
-Project updates should be left as comments in the project threads. Even if the
-update is "the project is blocked." or "No developments this week.", the updates
-should still be posted. This removes ambiguity for folks visiting the project,
-avoiding questions like "I don't see an update this week; is it being worked on
-or not?"
+For all active projects (anything that isn't in the `Not Started`, `Completed`,
+and `On Hold` statuses), fortnightly public updates are the best way for
+Openverse contributors and maintainers to understand the health of a project.
+Importantly, these updates allow us to allocate more time or material resources
+to projects that need them before work stalls completely. They also allow for
+identifying projects that have stalled and may need special attention, if they
+haven't seen an update in some time.
+
+Project updates should be left as comments in the project threads by the project
+lead or anyone working on a project. If you merge or a significant update to a
+project, or identify a bug that impacts an open project, please leave an update!
```

Comment:
Looks like a small editing error occurred here, suggested a fix for the typo and slight clarity.

```suggestion
lead or anyone working on a project. If you merge a significant PR related to the project or identify a bug that impacts an open project, please leave an update!
```